src/inc/clrnt.h

```diff
@@ -853,6 +853,35 @@ typedef union _UNWIND_CODE {
     USHORT FrameOffset;
 } UNWIND_CODE, *PUNWIND_CODE;
 
+// CFI_CODE is 8 byte struct regardless of operation for simplicity.
+// ObjectWriter will parse it and invoke a proper MCStream API in LLVM.
+typedef struct _CFI_CODE {
+    UCHAR CodeOffset;       // Offset from the start of code the frame covers.
+    UCHAR OpCode;           // Opcode encoded as Operation
+    SHORT DwarfReg;         // Dwarf register number. -1 is invalid.
+    INT Offset;             // Offset
+} CFI_CODE, *PCFI_CODE;
+
+// This is from LLVM MCDwarf.h
```

Comment:
Also, if you have copied this from LLVM - you need to be careful about doing the right thing wrt. license.